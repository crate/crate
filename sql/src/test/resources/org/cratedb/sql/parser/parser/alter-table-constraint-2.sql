ALTER TABLE t2 ADD CONSTRAINT P PRIMARY KEY (c3)