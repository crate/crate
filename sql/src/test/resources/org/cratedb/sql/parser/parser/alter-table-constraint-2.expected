org.cratedb.sql.parser.parser.AlterTableNode@5289cf1e
name: t2
statementType: ALTER TABLE
objectName: t2
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@557ce4e7

	[0]:	
	org.cratedb.sql.parser.parser.ConstraintDefinitionNode@7a982589
	constraintName: p
	constraintType: PRIMARY_KEY
	properties: null
	name: p
	elementType: AT_DROP_CONSTRAINT
	columnList: 		
		org.cratedb.sql.parser.parser.ResultColumnList@1ac88440

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@6262937c
		exposedName: c3
		name: c3
		tableName: null
		isDefaultColumn: false
		type: null